Gemfile

```diff
@@ -69,7 +70,6 @@ end
 
 group :test do
   gem 'capybara', '~> 3.39', '>= 3.39.2'
-  gem 'faker', '~> 3.2'
```

Comment:
We remove `Faker` gem but not update the existing factories, we want to remove it ?

Reply:
It was moved to dev, test group

Reply:
I didn't see it